src/fn/fn-slider.scss

```diff
@@ -48,6 +49,14 @@ $slider-height: 0.5rem;
     opacity: $fn-disabled-opacity;
   }
 
+  &__wrapper {
+    @include fn-reset();
+
+    position: relative;
+    margin-left: $slider-wrapper-offset;
+    margin-right: $slider-wrapper-offset;
```

Comment:
You can use the `fn-set-margins-x` mixin here

Reply:
No need since values are equal